templates/CRM/common/debug.tpl

```diff
@@ -8,26 +8,26 @@
  +--------------------------------------------------------------------+
 *}
 <!-- .tpl file invoked: {$tplFile}. Call via form.tpl if we have a form in the page. -->
-{if isset($smarty.get.smartyDebug|smarty:nodefaults)}
+{if $debugging.smartyDebug}
 {debug}
 {/if}
 
-{if isset($smarty.get.sessionReset|smarty:nodefaults)}
```

Comment:
@eileenmcnaughton if I passed in ?sessionReset=FALSE previously would that have been passed through you think?

Reply:
yeah - that's my understanding of the code - can't say I love it

Reply:
so we are now changing it to only if it was TRUE right? I guess its more a thing with the cacheCleanup and configReset where passing ?configReset=0 would have triggered the reset previously but now not now

Reply:
@seamuslee001 yeah - that sounds more like eliminating a bug than losing a feature to me

Reply:
hmm ok